packages/patternlab/styleguide/source/assets/js/modules/responsiveTable.js

```diff
@@ -213,141 +207,32 @@ export default (function (window, document, $) {
-        scrollLeft: (currentScrollLeft - amountToScroll) < 0 ? 0 : (currentScrollLeft - amountToScroll)
-      }, 250);
-    });
-
-    // Scrollbar right arrow.
-    $(".ma__table__horizontal-nav__right").click(function () {
-      const $scrollContainer = $(this).parents(".js-ma-responsive-table").find(".ma__table--responsive__wrapper");
-      // On click of left arrow element, animate the movement of the scrollbar
-      // to the left by the integer amount defined in `amountToScroll`.
-      $scrollContainer.animate({
-        scrollLeft: $scrollContainer.scrollLeft() + amountToScroll
-      }, 250);
-    });
-
-    // Scroll indicator element.
-    $(".ma__scroll-indicator").on("click", handleScrollerClick);
-
-    $(".ma__scroll-indicator__button")
-      .on("mousedown", handleScrollerInteraction)
-      // Deaden clicking on the scroll button
-      // in order to handle click on parent.
-      .on("click", e => e.stopPropagation());
-
-  }
-
-  // Calculate and set the widths of scrollbar elements.
-  function recalcScrollbar(rt) {
-    // Table width.
-    const tableWidth = rt.$table.width();
-    // Table container width.
-    const tableContainerWidth = rt.$table.parent().width();
-    // Width of the arrows: < and >.
-    const scrollArrowWidth = 16;
-    // Margin to the left or right of the arrows: < and >.
-    const scrollArrowMargin = 15;
-    // Scrollbar container width = table container width minus arrows and arrow margin.
-    const scrollbarContainerWidth = tableContainerWidth - (scrollArrowWidth + scrollArrowMargin) * 2;
-    // Scrollbar width.
-    const scrollbarWidth = scrollbarContainerWidth + scrollbarContainerWidth * (tableWidth - tableContainerWidth) / tableWidth;
-    // Button width.
-    const buttonWidth = scrollbarContainerWidth * tableContainerWidth / tableWidth;
-
-    const $scrollbar = rt.$root.find(".clip-scrollbar");
-    const $scrollbarIndicator = $scrollbar.find(".ma__scroll-indicator");
-
-    $scrollbar.find(".ma__scroll-indicator--bar").css("width", `${scrollbarWidth}px`);
-    $scrollbarIndicator.css("width", `${scrollbarContainerWidth}px`);
-    $scrollbarIndicator.find(".ma__scroll-indicator__button").css("width", `${buttonWidth}px`);
-  }
-
   // Allow a parameter to prevent triggering scrolling in an infinite loop.
   let skip = 0;
-
   // Calculate and set the scroll position of the other components when one component is scrolled.
   function handleTableScroll(e) {
     if (skip === 0) {
       const t = e.target;
       // No matter the element, the percentage as a decimal should be the amount of pixels scrolled
       // divided by the max amount that can be scrolled which is the scroll width minus the width.
       const scrollPercent = t.scrollLeft / (t.scrollWidth - t.offsetWidth);
-      ["ma__table--responsive__wrapper", "ma__scroll-indicator", "sticky-thead-wrapper"].map(scrollable => {
+      [
+        "ma__table--responsive__wrapper",
+        "ma__scroll-indicator",
```

Comment:
```suggestion
        "ma__scroll-indicator",
```

Do we still use this class?

Reply:
Thank you for mentioning that.  This is not in use and the class and related lines are cleaned up.